QgisModelBaker/libqgsprojectgen/generator/generator.py

```diff
@@ -129,10 +132,11 @@ def layers(self, filter_layer_list=[]):
             is_attribute = bool(record.get("attribute_name"))
             is_structure = record.get("kind_settings") == "STRUCTURE"
             is_nmrel = record.get("kind_settings") == "ASSOCIATION"
-            is_basket_table = (
+            # when the basket handling is not considered we do not consider tables named as basket tables consider as basket tables
```

Comment:
Is this right? The double `consider` makes it difficult to understand.

Reply:
Even confusing for me now. But it's correct. I will change the comment.